back/exercises1.sh

```diff
@@ -0,0 +1,29 @@
+# !/bin/bash
+echo -n "年齢（半角数字で入力）=> "
+read age
+echo -n "性別（man, womanで入力）=> "
+read gender
+if test -z $gender || [ -z $age ] ; then  # 空白時
+  echo "exit 1" # 空白時
```

Comment:
```suggestion
  exit 1
```
echoは不要ですね！
他同様箇所あり